presto-main/src/main/java/com/facebook/presto/execution/SqlTaskManager.java

```diff
@@ -143,7 +143,7 @@ public QueryContext load(QueryId key)
                         maxQueryMemoryPerNode,
                         localMemoryManager.getPool(LocalMemoryManager.GENERAL_POOL),
                         localMemoryManager.getPool(LocalMemoryManager.SYSTEM_POOL),
-                        taskNotificationExecutor,
+                        taskManagementExecutor,
```

Comment:
I would prefer we convert `taskNotificationExecutor` to be a scheduled thread pool.  I think of the *timeout* as a form of notification, so this fits the existing naming and usage.  Additionally, the existing pool is already a fix size and has a configuration property to control the size.